aten/src/ATen/core/Formatting.cpp

```diff
@@ -295,13 +295,15 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
         stream << ", axis: " << tensor_.q_per_channel_axis();
       }
     }
-    if (tensor._has_fw_grad()) {
+
+    // Proxy check for if autograd was built
+    if (tensor.getIntrusivePtr()->autograd_meta()) {
       auto& fw_grad = tensor._fw_grad(/* level */ 0);
       if (fw_grad.defined()) {
```

Comment:
Is this check still necessary?

Reply:
Yes because we don't want to print all the undefined Tensors you get for things that don't have a forward grad.